openml-datarobot/README.md

```diff
@@ -15,9 +15,9 @@ To import DataRobot models you need to place them in a special folder structure
 └── model.json
 ```
 
-\<model name\> can be any name you want to give to the folder containing the model. You will need to provide the path to this folder.
+\<model name\> can be any name you want to give to the folder containing the model. At the import, you will need to provide the path to this folder.
 
-"model.json" is an optional JSON file describing the schema of the model. It needs to be in the following format:
+"model.json" is an optional JSON file describing the schema of the model. The following is an example of the required format:
```

Comment:
we should stop replicating the example format in every provider and keep it centralized in openml-api repo + linking in each provider, otherwise someone is going to suffer a lot in the near future

Reply:
I like the idea, if the file structure is prone to change. 
Creating the copy doesn't cost anything, but maintaining it on different places can be a pain.
Anyone has any thoughts on how often the json structure might change?